### PR TITLE
New package: InterstellarResearch.Daqarta version 11.7.0

### LINKED ISSUE #113522: [Package Request]: Daqarta
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### What's the type of your software?

exe

### Please provide the following information

Download Page Url: https://www.daqarta.com/dqdown.htm
Install Options: 
Publisher: Interstellar Research
Publisher Page: https://www.daqarta.com
Package Page: https://www.daqarta.com
Package Name: Daqarta
Description: 
Package Version: 11.7.0
Installer URL: https://www.daqarta.com/DHsetup.exe
License: Proprietary
License Page: 
Copyright: © Copyright 2007 - 2023 by Interstellar Research

### PR DESCRIPTION
**Pull request by CoolPlayLin**
### Pull request has been created with [Komac](https://github.com/russellbanks/Komac) v1.10.1 :rocket:
- Resolve  #113522
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/117685)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by Trenly** [comment]
Even after switching to nullsoft installer type, still is not unattended
image

File: manifests/i/InterstellarResearch/Daqarta/11.7.0/InterstellarResearch.Daqarta.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created with Komac v1.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: InterstellarResearch.Daqarta
+PackageVersion: 11.7.0
+InstallerType: burn
```
Comment (AsciiWolf) [inline manifests/i/InterstellarResearch/Daqarta/11.7.0/InterstellarResearch.Daqarta.installer.yaml:6]:
```suggestion
InstallerType: nullsoft
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by CoolPlayLin** [comment]
No Closing

**Pull request closed without merging**